src/components/editor/RDFModal.jsx

```diff
@@ -12,7 +12,8 @@ class RDFModal extends Component {
   }
 
   render() {
-    const json = JSON.parse(this.props.linkedData)
+    // const json = JSON.parse(this.props.linkedData)
```

Comment:
remove old?

Reply:
Leaving for now; I think this will be needed soon.